app/scripts/controllers/sign.ts

```diff
@@ -105,7 +105,6 @@ export type SignControllerOptions = {
   messenger: SignControllerMessenger;
   keyringController: KeyringController;
   preferencesController: PreferencesController;
-  sendUpdate: () => void;
```

Comment:
It doesn't look like this shouldn't be part of the options.

Reply:
Good spot, an old argument that used to be needed to force a state update.